samples/unity-of-bugs/Assets/Scripts/SmokeTester.cs

```diff
@@ -48,21 +48,7 @@ public void Start()
             arg = intent.Call<String> ("getStringExtra", "test");
         }
 #elif UNITY_IOS
-        string pListPath = Path.Combine(Application.dataPath.Substring(0, Application.dataPath.LastIndexOf('/')), "Info.plist");
-        using (var streamReader = new StreamReader(pListPath, true))
-        {
-            var rawPlist = streamReader.ReadToEnd();
-            var key = "RunSentrySmokeTest";
-            if (rawPlist.Contains(key))
-            {
-                arg = "smoke";
-                Debug.Log("Key " + key + " found on Info.plist starting Smoke test.");
-            }
-            else
-            {
-                Debug.Log("To run Smoke Test, please add key " + key + " into Info.plist");
-            }
-        }
+        arg = getTestArgObjectiveC();
```

Comment:
Does `Environment.GetCommandLineArgs();` return already the same thing? Unity might have done data binding here

Reply:
I've tried and it doesn't work. I'll add a comment